app/javascript/components/App.js

```diff
@@ -1,23 +1,67 @@
 import React, { Component } from "react";
-import PatientIndex from "./pages/PatientIndex";
+import {
+  BrowserRouter as Router,
+  NavLink,
+  Route,
+  Switch,
+} from "react-router-dom";
+import Header from "./components/Header";
+import Home from "./pages/Home";
+import PatientInfo from "./pages/PatientInfo";
 import "./App.css";
 
 export default class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      patients: [],
-      meds: [],
+      patientsArray: [],
     };
-    fetch("/patients/2")
-      .then((resp) => resp.json())
-      .then((patient) => this.setState({ meds: patient.medications }));
   }
+
+  componentDidMount() {
+    this.readPatients();
+  }
+
+  readPatients = () => {
+    fetch("/patients")
+      .then((resp) => resp.json())
+      .then((patients) => this.setState({ patientsArray: patients }))
+      .catch((errors) => console.log("Patients errors:", errors));
+  };
+
   render() {
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route,
+    } = this.props;
     return (
-      <div className="app">
-        <PatientIndex meds={this.state.meds} />
-      </div>
+      <>
+        <Router>
+          {/* <Header {...this.props} /> */}
```

Comment:
If you aren't using this code you should remove it. It is a best practice to not have commented out code in the main branch.

Reply:
Sarah this is for later.